### PR TITLE
Feat/4266  sub content types

### PR DESCRIPTION
**Pull request by asf-algoo**
See #4266.

:warning: don't forget to put the migration revisions with #4278 in sequence.

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [ ] Manual, quality tests have been done: will be tested in #4256 (the corresponding frontend development).

### CODE REVIEW

File: backend/tracim_backend/migration/versions/ae30497b6168_taking_care_of_moved_contents_properties.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import json
+
+from alembic import op
+import sqlalchemy as sa
+
+from tracim_backend.models.meta import metadata
+
+# revision identifiers, used by Alembic.
+revision = "ae30497b6168"
+down_revision = "78a01733957f"
```
Comment (inkhey) [inline backend/tracim_backend/migration/versions/ae30497b6168_taking_care_of_moved_contents_properties.py:17]: Depending which one of this or https://github.com/tracim/tracim/pull/4278 is merged first, we do need to adapt the down revision in one of those two PR.

File: backend/tracim_backend/migration/versions/ae30497b6168_taking_care_of_moved_contents_properties.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
+"""taking care of moved contents properties
```
Comment (inkhey) [inline backend/tracim_backend/migration/versions/ae30497b6168_taking_care_of_moved_contents_properties.py:1]: A bit more explanation may be nice.

**Review by inkhey — APPROVED** [review]

File: backend/tracim_backend/migration/versions/ae30497b6168_taking_care_of_moved_contents_properties.py (hunk 1 above)
Comment (grignards) [reply backend/tracim_backend/migration/versions/ae30497b6168_taking_care_of_moved_contents_properties.py:23]: Yes, I thought about it.

File: backend/tracim_backend/migration/versions/ae30497b6168_taking_care_of_moved_contents_properties.py (hunk 2 above)
Comment (grignards) [reply backend/tracim_backend/migration/versions/ae30497b6168_taking_care_of_moved_contents_properties.py]: Done.

**Pull request merged**